test/test_LO.py

```diff
@@ -0,0 +1,131 @@
+import torch
+from torch.testing._internal.common_utils import TestCase, run_tests
+from torch.utils._pytree import tree_map
+
+from typing import Iterator, List
+import functools
+import contextlib
+
+# TODO: move this into library proper
+@contextlib.contextmanager
+def no_dispatch() -> Iterator[None]:
```

Comment:
Can you send in a patch for this? `torch.overrides` is probably the most logical spot to put it. Will need a docblock too.